src/orquestra/sdk/_base/cli/_arg_resolvers.py

```diff
@@ -438,7 +438,7 @@ def resolve_limit(
                     "Enter maximum number of results to display. "
                     "If 'None', all results will be displayed."
                 ),
-                default="None",
+                default=None,
```

Comment:
I just saw it mentioned in the tests. IIRC something's wrong with the overall typing here, as "None" should be an accepted output (as much as I don't like this)

Reply:
If I wasnt mistaken, the correct type here is Optional[int], so no idea how that would work with string type passed out tbh.
None type seems to be better anyway